test/index.js

```diff
@@ -5,8 +5,8 @@ import Adapter from 'enzyme-adapter-react-16';
 
 Enzyme.configure({ adapter: new Adapter() });
 
-function assertLength(length) {
-  return function $assertLength(selector) {
+const assertLength = length =>
```

Comment:
i think it's slightly nicer to have normal functions at top level... shades of https://github.com/airbnb/javascript/tree/master/react#class-vs-reactcreateclass-vs-stateless in preferring normal functions over arrows